newrelic/resource_newrelic_alert_muting_rule_test.go

```diff
@@ -44,6 +96,66 @@ func TestAccNewRelicAlertMutingRule_Basic(t *testing.T) {
 	})
 }
 
+func TestAccNewRelicAlertMutingRule_WithSchedule(t *testing.T) {
+	resourceName := "newrelic_alert_muting_rule.bar"
+	rName := acctest.RandString(5)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckNewRelicAlertMutingRuleDestroy,
+		Steps: []resource.TestStep{
+			// Test: Create
+			{
+				Config: testAccNewRelicAlertMutingRuleWithSchedule(
+					rName,
+					"new muting rule",
+					"product",
+					"EQUALS",
+					"APM",
+					`
+						start_time         = "2021-01-21T15:30:00"
+						end_time           = "2021-01-21T16:30:00"
+						time_zone          = "America/Los_Angeles"
+						repeat             = "WEEKLY"
+						end_repeat         = "2022-06-11T12:00:00"
+						weekly_repeat_days = ["MONDAY", "TUESDAY", "FRIDAY"]
+					`,
+				),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckNewRelicAlertMutingRuleExists(resourceName),
+				),
+			},
+			//Test: Update
+			{
+				Config: testAccNewRelicAlertMutingRuleWithSchedule(rName,
+					"updated muting rule schedule",
+					"conditionType",
+					"NOT_EQUALS",
+					"baseline",
+					`
+						start_time         = "2021-02-21T15:30:00"
+						end_time           = "2021-02-21T16:30:00"
+						end_repeat         = "2022-06-11T12:00:00"
+						repeat             = "DAILY"
+						time_zone          = "America/Los_Angeles"
+					`,
+				),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckNewRelicAlertMutingRuleExists(resourceName),
+				),
+			},
+			//Test: Import
+			// TODO - determine why this is failing
+			//	{
+			//		ResourceName:      resourceName,
+			//		ImportState:       true,
+			//		ImportStateVerify: true,
+			//	},
```

Comment:
What is this import test testing?

Reply:
terraform can import existing resources - say ones that have been defined via the UI, but you want to take over management with via TF. Import queries an existing resource and then adds it to state after diffing it with how you have it represented in the config. 
Basically, you create a config that reflects an existing rule, then from the CLI run `terraform import newrelic_muting_rule.bar 400304:123456` and it checks the config for `bar` vs the identified rule and if they line up, saves it into state

Reply:
Ah, I see.  Does this test pass if you uncomment it out?  Or is there still implementation left to do in this PR to enable import?

Reply:
We just ran this, and it passed! 🎉